cocos/tween/tween.ts

```diff
@@ -93,6 +107,98 @@ export class Tween<T extends object = any> {
         return this;
     }
 
+    /**
+     * @en Return a new Tween instance which reverses all actions in current tween
+     * @zh 返回新的缓动实例，其会翻转当前缓动中的所有动作
+     * @return @en The new tween instance which reverses all actions in current tween @zh 新的缓动实例，其会翻转当前缓动中的所有动作
+     * @note @en The returned tween instance is a new instance which is not the current tween instance.
+     *       @zh 返回的缓动实例是新的生成的实例，并不是当前缓动实例
+     */
+    reverse (): Tween<T>;
+
+    /**
+     * @en Reverse an action by ID in the current tween
+     * @zh 翻转当前缓动中特定标识的动作
+     * @param id @en The ID of the internal action in the current tween to reverse @zh 要翻转的当前缓动中的动作标识
+     * @return @en The current tween instance @zh 当前缓动实例
+     * @note @en Don't use -1 for id since -1 is the default value for internal actions
+     *       @zh 请不要使用 -1 作为标识，因为 -1 是内部动作标识的默认值
+     */
+    reverse (id: number): Tween<T>;
+
+    /**
+     * @en Reverse an action by ID in a specific tween
+     * @zh 翻转特定缓动中特定标识的动作
+     * @param otherTween @en The tween in which to find the action by ID
+     *                   @zh 根据标识在关联的缓动中查找动作
+     * @param id @en The ID of the action to reverse @zh 要翻转的动画标识
+     * @return @en The current tween instance @zh 当前缓动实例
+     * @note @en If `id` is not assigned or its value is -1, it will reverse all actions in the `otherTween`
+     *       @zh 如果 `id` 没有被指定或者值为 -1，那么将翻转 `otherTween` 中所有的动作
+     */
+    reverse<U extends object = any> (otherTween: Tween<U>, id?: number): Tween<T>;
+
+    reverse<U extends object = any> (otherTweenOrId?: Tween<U> | number, id?: number): Tween<T> {
+        if (otherTweenOrId == null && id == null) {
+            if (this._actions.length === 0) {
+                warn('reverse: current tween could not be reversed, empty actions');
+                return this.clone(this._target as T);
```

Comment:
Is it better to dummy tween action as the comment says.

Reply:
If `this._action.length` is null, Tween.clone will return an empty Tween.